packages/chakra-components/src/ComboBox/useSelect.js

```diff
@@ -74,8 +74,11 @@ function useHoistedState(initialState, reducer) {
   return [state, setState];
 }
 
+let rapidTypeaheadDebouncer = null;
```

Comment:
This can be used with the way the other debouncer has  been used ?

Reply:
Again, this code was written by you, appeared in my PR after rebasing.
![image](https://user-images.githubusercontent.com/39694575/82174288-ce0f4600-98ed-11ea-9550-ce8081275229.png)
Haven't gone through on why this was used.